app/controllers/preview_controller.rb

```diff
@@ -1,3 +1,8 @@
 class PreviewController < ApplicationController
   layout 'metadata_presenter/application'
+
+  def service
+    @service ||= MetadataPresenter::Service.new(service_metadata, editor: editable?)
+  end
+  helper_method :service
```

Comment:
Maybe we can delete this line as there is no difference from the application controller

Reply:
I will see here locally